e2e-win/tool_stub.Tests.ps1

```
@@ -0,0 +1,60 @@
+Describe 'tool-stub' {
+    BeforeAll {
+        $originalPath = Get-Location
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'originalPath' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)